### DISCUSSION TITLE
How to achieve functionality similar to WPF's ClipGeometry?

### DISCUSSION THREAD

**Discussion by lindexi** [unoplatform/uno#17279] [Q&A]
I am currently developing an application similar to Photoshop, which has the feature of erasing elements based on the Path. This requires any element control to be erasable. In WPF, I can easily use ClipGeometry to achieve this. However, I am now using the UNO framework and I haven't found an alternative method. Is this functionality missing in UNO?

**Comment by jeromelaban** [comment]
The Composition API supports geometry clipping, you may be able to achieve such a result with it, depending on what you want to clip.

**Reply by lindexi** [reply]
Thank you @jeromelaban . And I will try it.
